cluster-logging.md

```diff
@@ -54,32 +54,32 @@ The "Location" column can be one of the following:
+| `kubernetes.namespace_name` | resource | container | required stream label | **(DEPRECATED)** Compatibility attribute, contains same information as `k8s.namespace.name` |
 | `kubernetes.pod_name` | resource | container | stream label | **(DEPRECATED)** Compatibility attribute, contains same information as `k8s.pod.name` |
-| `openshift.cluster_id` | resource | all | stream label | **(DEPRECATED)** Compatibility attribute, contains same information as `openshift.cluster.uid` |
-| `openshift.cluster.uid` | resource | all | stream label | |
-| `openshift.log.source` | resource | all | stream label | |
-| `openshift.log.type` | resource | all | stream label | |
-| `openshift.labels.*` | resource | all | stream label | |
+| `openshift.cluster_id` | resource | all | | **(DEPRECATED)** Compatibility attribute, contains same information as `openshift.cluster.uid` |
+| `openshift.cluster.uid` | resource | all | required stream label | |
+| `openshift.log.source` | resource | all | required stream label | |
+| `openshift.log.type` | resource | all | required stream label | |
+| `openshift.labels.*` | resource | all | structured metadata | |
 | `k8s.node.name` | resource | all | stream label | |
-| `k8s.namespace.name` | resource | container | stream label | |
+| `k8s.namespace.name` | resource | container | required stream label | |
 | `k8s.container.name` | resource | container | stream label | |
 | `k8s.pod.labels.*` | resource | container | structured metadata | |
 | `k8s.pod.name` | resource | container | stream label | |
-| `k8s.pod.uid` | resource | container | stream label | |
-| `k8s.cronjob.name` | resource | container | structured metadata | Conditionally forwarded based on creator of Pod |
-| `k8s.daemonset.name` | resource | container | structured metadata | Conditionally forwarded based on creator of Pod |
-| `k8s.deployment.name` | resource | container | structured metadata | Conditionally forwarded based on creator of Pod |
-| `k8s.job.name` | resource | container | structured metadata | Conditionally forwarded based on creator of Pod |
+| `k8s.pod.uid` | resource | container | structured metadata | |
+| `k8s.cronjob.name` | resource | container | stream label | Conditionally forwarded based on creator of Pod |
+| `k8s.daemonset.name` | resource | container | stream label | Conditionally forwarded based on creator of Pod |
+| `k8s.deployment.name` | resource | container | stream label | Conditionally forwarded based on creator of Pod |
+| `k8s.job.name` | resource | container | stream label | Conditionally forwarded based on creator of Pod |
 | `k8s.replicaset.name` | resource | container | structured metadata | Conditionally forwarded based on creator of Pod |
 | `k8s.statefulset.name` | resource | container | structured metadata | Conditionally forwarded based on creator of Pod |
```

Comment:
I think we forgot to make these stream labels as well? Do you agree @periklis ?

Reply:
Seems to make sense to also make `k8s.statefulset.name` a stream label 👍

Reply:
Let's get them in then.

Reply:
Done